### PR TITLE
[Calling] Release note for PPTLive  GA

### PR DESCRIPTION
**Pull request by ShaunaSong**
# What

 Release note for PPTLive  GA

We are excited to announce that the Azure Communication Services Web UI Library now can view PowerPoint Live sessions initiated by a Teams client. Users can follow along with the current slide the presenter is sharing and view presenter annotations. Developers can use this functionality today through our composites (e.g CallComposite, CallWithChatComposite) as well as through components (e.g VideoGallery).

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

**Review by Mohtasim — APPROVED** [review]
Congratulations. ^_^

**Review by dmceachernmsft — APPROVED** [review]

File: change-beta/@azure-communication-react-b4bd7243-4950-4fee-b9c5-6c42c93b84a4.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "prerelease",
+  "area": "feature",
+  "workstream": "PowerPoint Live - Teams Interop",
+  "comment": "We are excited to announce that the Azure Communication Services Web UI Library now can view PowerPoint Live sessions initiated by a Teams client. Users can follow along with the current slide the presenter is sharing and view presenter annotations. Developers can use this functionality today through our composites (e.g CallComposite, CallWithChatComposite) as well as through components (e.g VideoGallery).",
+  "packageName": "@azure/communication-react",
+  "email": "93549644+ShaunaSong@users.noreply.github.com",
+  "dependentChangeType": "patch"
```
Comment (alkwa-msft) [inline change-beta/@azure-communication-react-b4bd7243-4950-4fee-b9c5-6c42c93b84a4.json:8]: you need to change this to pre-release

Comment (dmceachernmsft) [reply change-beta/@azure-communication-react-b4bd7243-4950-4fee-b9c5-6c42c93b84a4.json:8]: is this going to GA? I would argue its a `minor` change if so.

Comment (alkwa-msft) [reply change-beta/@azure-communication-react-b4bd7243-4950-4fee-b9c5-6c42c93b84a4.json:8]: this is what we have in our documentation

image

**Review by JamesBurnside — APPROVED** [review]

File: change-beta/@azure-communication-react-b4bd7243-4950-4fee-b9c5-6c42c93b84a4.json (hunk 1 above)
Comment (JamesBurnside) [reply change-beta/@azure-communication-react-b4bd7243-4950-4fee-b9c5-6c42c93b84a4.json:8]: This looks like its just for inProgress->beta, we should have different instructions for beta->stable

**Comment by JamesBurnside** [comment]
Added breaking-change label and verified no customer impact breaking change. Change is the new pptLive property (that is technically required) on the callState - it is OK to be required as this interface is not implemented by Contoso

**Pull request merged**